releasing.md

```diff
@@ -0,0 +1,29 @@
+**Before Release**
```

Comment:
WDYT about turning this into an issue template?

Reply:
#743